### PR TITLE
7.0 sale mrp sub state

### PR DESCRIPTION
**Pull request by chafique-delli**
This module allows to complete the sub state of the sale order
with the different stages of production.

### CODE REVIEW

**Comment by eLBati** [comment]
This depends on https://github.com/OCA/sale-workflow/pull/93

File: sale_mrp_sub_state/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    'name': 'Sale Mrp Sub State',
+    'version': '0.1',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'description': """
+    This module allows to complete the sub state of the sale order
+    with the different stages of production.
+    """,
+    'author': 'Akretion',
```
Comment (gurneyalex) [inline sale_mrp_sub_state/__openerp__.py:31]: please add "Odoo Community Association (OCA)" to the authors list.

**Comment by gurneyalex** [comment]
could you please add an automated test?

**Comment by pedrobaeza** [comment]
Closing as this seems abandoned without repairing comments. If not, please feel free to reopen.

**Pull request closed without merging**